heat-exchanger/tutorial.yaml

```
@@ -0,0 +1,8 @@
+name: heat-exchanger
+couples: 
```

```suggestion
participants: 
```
could be an alternative?

---

totally! This is more in agreement with the naming scheme i guess :)